frontend/src/util/editorCommentUtils.js

```diff
@@ -43,6 +42,25 @@ function isCommentThreadIDMark(mayBeCommentThread) {
     return mayBeCommentThread.indexOf(COMMENT_THREAD_PREFIX) === 0;
 }
 
+export function insertMaybeComment (editor, selectedText, setMaybeComment) {
+    setMaybeComment(selectedText);
+    Editor.addMark(editor, 'isMaybeComment', true);
```

Comment:
can you pull `isMaybeComment` out into a constant and stick it in `constants.js` or `redactionUtils` or wherever the other prefixes are